openml/datasets/functions.py

```diff
@@ -333,27 +333,40 @@ def _load_features_from_file(features_file: str) -> Dict:
         return xml_dict["oml:data_features"]
 
 
-def check_datasets_active(dataset_ids: List[int]) -> Dict[int, bool]:
+def check_datasets_active(
+    dataset_ids: List[int],
+    raise_error_if_not_exist: bool = True,
+) -> Dict[int, bool]:
     """
     Check if the dataset ids provided are active.
 
+    Raises an error if a dataset_id in the given list
+    of dataset_ids does not exist on the server.
+
     Parameters
     ----------
     dataset_ids : List[int]
         A list of integers representing dataset ids.
+    raise_error_if_not_exist : bool, optional (default=True)
+        Flag that if activated can raise an error, if one or more of the
+        given dataset ids do not exist on the server.
 
     Returns
     -------
     dict
         A dictionary with items {did: bool}
     """
-    dataset_list = list_datasets(status="all")
+    dataset_list = list_datasets(
+        dataset_ids=dataset_ids,
```

Comment:
```suggestion
        data_id=dataset_ids,
```

Reply:
Actually this is a rather important one that hadn't caught my eye first time around 😅

Reply:
Strange, I got a suggestion for that and the function had no argument like that at all :S. Nice catch.

Reply:
like because of the `**kwargs` (used to allow all filters)